src/main/java/edu/kis/powp/jobs2d/drivers/adapter/LineDriverAdapter.java

```diff
@@ -25,14 +26,15 @@ public LineDriverAdapter(DrawPanelController drawController, ILine line, String
 	public void setPosition(int x, int y) {
 		this.startX = x;
 		this.startY = y;
+		new CommandGenerator().setPosition(x,y);
```

Comment:
LineDriverAdapter should not be modified. CommandGenerator could be a decorator to support "extending" other drivers with commands generator feature.